### PR TITLE
Use plural words when bulk operating multiple incidents

### PR DESCRIPTION
**Pull request by stveit**
Fixes parts of #423 
also removes some trailing whitespace

Uses spread operator to only override the text if multiple incidents are selected. Is this a clean way to do it?

### CODE REVIEW

**Review by podliashanyk — CHANGES_REQUESTED** [review]
Good grasp on spread operator! :) 
But I would suggest refactoring of the _CreateAckSignOffAction.tsx_ and _ManualCloseSignOffAction.tsx_ instead, and making them aware of whether they are being used for bulk operation or not. This is because there are other props that should also be updated for the bulk components:

- in CreateAck - _question_
- in ManualClose - _dialogTitle_, _question_, _title_ (maybe)

So the suggested solution is to add a new prop to CreateAck and ManualClose (something like isBulk) and update the components from there.

**Review by hmpf — APPROVED** [review]
Will merge when @podliashanyk is happy.

File: src/components/incident/CreateAckSignOffAction.tsx [hunk 1]
```diff
@@ -35,19 +36,29 @@ const CreateAck: React.FC<CreateAckPropsType> = ({ onSubmitAck, signOffActionPro
...
+    isDialogInputRequired:false,
+    dialogInputType: "text",
+    title: "Submit acknowledment",
+    question: "Are you sure you want to acknowledge this incident?",
+    onSubmit: handleSubmit,
+  }
+
+  const signOffActionPluralProps = {
+    dialogContentText: "Write a message describing why the incidents were acknowledged",
+    question: "Are you sure you want to acknowledge these incident?",
```
Comment (podliashanyk) [inline src/components/incident/CreateAckSignOffAction.tsx:55]:
```suggestion
    question: "Are you sure you want to acknowledge these incidents?",
```

File: src/components/incident/ManualCloseSignOffAction.tsx [hunk 2]
```diff
@@ -29,23 +30,35 @@ const ManualClose: React.FC<ManualClosePropsType> = ({
...
+    dialogInputType: "text",
+    title: "Manually close incident",
+    question: "Are you sure you want to close this incident?",
+    onSubmit: onManualClose,
+  }
+
+  const signOffActionPluralProps = {
+    dialogTitle: "Manually close incidents",
+    dialogContentText: "Write a message describing why the incidents were manually closed",
+    question: "Are you sure you want to close these incident?",
```
Comment (podliashanyk) [inline src/components/incident/ManualCloseSignOffAction.tsx:54]:
```suggestion
    question: "Are you sure you want to close these incidents?",
```

**Review by podliashanyk — CHANGES_REQUESTED** [review]
Good job so far!

Only updates for bulk re-open operation that remain (_title_, _question_)

File: src/components/incident/ManualCloseSignOffAction.tsx [hunk 3]
```diff
@@ -29,37 +30,60 @@ const ManualClose: React.FC<ManualClosePropsType> = ({
...
       <ConfirmationButton
-        title={"Reopen incident"}
-        question={"Are you sure you want to reopen this incident?"}
-        onConfirm={onManualOpen}
-        ButtonComponent={ButtonComponent}
+        className={classes.dangerousButton}
         buttonProps={{
-            variant: "contained"
+            variant: "contained",
         }}
```
Comment (stveit) [inline src/components/incident/ManualCloseSignOffAction.tsx:84]: When i tried to but `buttonProps` inside `reopenButtonDefaultProps` I got this error:

```
Type '{ children: string; color?: "inherit" | "primary" | "secondary" | "default" | undefined; disabled?: boolean | undefined; disableElevation?: boolean | undefined; disableFocusRipple?: boolean | undefined; ... 284 more ...; buttonProps: { ...; }; }' is not assignable to type '{ title: string; question: string; ButtonComponent?: "abbr" | "address" | "article" | "aside" | "b" | "bdi" | "bdo" | "big" | "blockquote" | "button" | "caption" | "cite" | "code" | "data" | ... 44 more ... | undefined; buttonProps?: Partial<...> | undefined; }'.
 The types of 'buttonProps.variant' are incompatible between these types.
  Type 'string' is not assignable to type '"text" | "outlined" | "contained" | undefined'.  TS2322
```

So i just left it where it was

Comment (podliashanyk) [reply src/components/incident/ManualCloseSignOffAction.tsx:84]: buttonProps is a type defined by MUI, but no worries we don't actually need to fix this

**Review by podliashanyk — APPROVED** [review]
Very well!

**Pull request merged**